department-of-veterans-affairs/caseflow
# Send to VACOLS flag

Connects #6520 

### Description
This PR adds a flag to upload to VACOLS. The flag is only set when the user hits 'Confirm Upload' and is unset when the hearings are uploaded to VACOLS.

### Testing Plan
1. Confirm the normal route of uploading hearings still works. 
1. Create a second schedule period, but cancel on the 'Review Assignments' page
1. Navigate to the 'Build Schedule' homepage and confirm no hearings are uploaded to VACOLS
